x/ibc/gmp/ibc_middleware.go

```diff
@@ -162,10 +163,6 @@ func (im IBCMiddleware) OnTimeoutPacket(
 	return im.app.OnTimeoutPacket(ctx, packet, relayer)
 }
 
-func (im IBCMiddleware) appOnRecvPacket(
-	ctx sdk.Context,
-	packet channeltypes.Packet,
-	relayer sdk.AccAddress,
-) ibcexported.Acknowledgement {
+func (im IBCMiddleware) appOnRecvPacket() ibcexported.Acknowledgement {
```

Comment:
@RedBird96 curious if this change in args breaks any sort of interface

Reply:
As long as we don't have any special analyzing logic in the Recv stage, it's okay.